R/git.R

```diff
@@ -125,60 +125,121 @@ git_blame <- function(git_repo_path,flags,commit_hash,file_path){
 git_create_sample_log <- function(folder_path="/tmp"){
   # Expand paths (e.g. "~/Desktop" => "/Users/someuser/Desktop")
   folder_path <- path.expand(folder_path)
-  folder_path <- file.path(folder_path,"kaiaulu_sample")
+  #folder_path <- file.path(folder_path,"kaiaulu_sample")
+  folder_path <- io_create_folder("kaiaulu_sample")
 
   #mkdir path/to/folder/sample
-  error <- system2('mkdir',
-                   args = c(folder_path),
-                   stdout = TRUE,
-                   stderr = FALSE)
+  #error <- system2('mkdir',
+  #                 args = c(folder_path),
+  #                 stdout = TRUE,
+  #                 stderr = FALSE)
 
 
   file_path <- file.path(folder_path,"hello.R")
 
+  io_make_sample_file(file_path, "print('hello world!')")
+
   #echo "print('hello world!')" >  path/to/folder/hello.R
-  error <- system2('echo',
-                   args = c("\"print('hello world!')\"",
-                            '>',
-                            file_path),
-                   stdout = TRUE,
-                   stderr = FALSE)
+  #error <- system2('echo',
+  #                 args = c("\"print('hello world!')\"",
+  #                          '>',
+  #                          file_path),
+  #                 stdout = TRUE,
+  #                 stderr = FALSE)
 
-  # git init path/to/folder
-  error <- system2('git',
-                   args = c('init',
-                            folder_path),
-                   stdout = TRUE,
-                   stderr = FALSE)
+  git_init(folder_path)
 
   git_repo <- file.path(folder_path,'.git')
 
-  # git --git-dir sample/.git --work-tree sample add hello.R
+  git_add(git_repo, folder_path, file_path)
+
+  # WILL UNCOMMENT THIS BELOW LINE AFTER TEST
```

Comment:
Should remove these before sending the Pull Request, or they may be accidentally accepted into the main code.